website/versioned_docs/version-1.0/advanced/override_grammar/basic.md

```diff
@@ -142,6 +151,20 @@ Note that trailing and leading whitespace are ignored, the above is equivalent t
 $ python my_app.py 'msg=    hello world    '
 ```
 
+### Escaped characters in unquoted values
+If you really intend to have trailing or leading whitespaces in unquoted values, it is still possible by escaping them.
+For instance in the following example `msg` will be set to the string `" test "`:
+
+```shell
+$ python my_app.py 'msg=\ test\ '
+```
+
+Here is the full list of characters that may be escaped in unquoted values:
+* Space (`\ `) (for trailing / leading whitespaces as in the example above)
+* Tab (`\\t`) (for trailing / leading tabs, though note that passing tabs through your shell can be [tricky](https://stackoverflow.com/questions/6392249/how-to-enter-a-tab-char-on-command-line))
+* Comma (`\,`) (to avoid confusion with the sweeps described below)
+* Backslash (`\\`) (to allow for an actual backslash preceding any of the above characters, e.g. a sweep value ending with a backslash)
+
```

Comment:
This is too verbose, but I don't want to suggest edits before we decide on supporting escaping of other characters.

Reply:
I'll give it a shot while adding more escape characters. I just realized though that there are two `basic.md` files and they seem to be slightly out-of-sync (`website/docs/advanced/override_grammar/basic.md` and `website/versioned_docs/version-1.0/advanced/override_grammar/basic.md`), even in master. Do you know which one is currently the "official" version?

Reply:
Doc updated in 8d65ff1a

Reply:
> Update the one in docs (that would appear in https://www.internalfb.com/intern/staticdocs/hydra/docs/next/).

Got it. I reverted my changes to versioned docs in 04a53d8aa